### ISSUE TITLE
Lizmap administration Error 500 - impossible to save any setting

### ISSUE THREAD

**Issue by news24lor** [3liz/lizmap-web-client#713]
Hi Lizmap

I have installed last lizmap version (3.2pre171102) and last QGIS (2.18.14) on Mac OS X and it works. The Montpellier Demo works and it's fast.
The first problem is in the Admin Setting. I enter with Admin, Admin and I see anything but it's impossible to create, modify, change any value because when I select SAVE I have this error:

Error 500. A technical error has occured. Sorry for this trouble.

There are two log in lizmap/var/log
In error.log there is this row at the end

2017-11-20 15:25:23	xxx.xxx.xxx.xxx	error	2017-11-20 15:25:23	[403]	query not valid (attempt to write a readonly database (UPDATE jlx_user SET 
 usr_password= '$2y$10$ZXucYNvCy9IfVCdnKlDiU.TeDKQTP9WOXZHeo0uDyb0FMpFZx9.N2' WHERE  usr_login  = 'admin'))	/Applications/MAMP/htdocs/mylizmap/lib/jelix/plugins/db/sqlite3/sqlite3.dbconnection.php	123

Why a readonly database?

Thanks

**Comment by news24lor** [comment]
After many check I have resolved this problem
I have changed the permission of lizmap/var/db  folder to 777
and I change the permission of lizmap/var/config/lizmapConfig.ini.php file to 777

Now it works

Thanks

**Comment by mdouchin** [comment]
Hi, please never use 777 for any folder !
Have you seen the INSTALL.md file here ?
https://github.com/3liz/lizmap-web-client/blob/master/INSTALL.md

There is a command dedicated to set the correct rights on folders, only when needed

**Comment by news24lor** [comment]
Yes I know but I have follow the instruction and I have set the permission with the script. The only difference in Mac OS X is the user and group: 'www-data' in Debian, '_www' in OS X.
I have the problem after when I want to save a modify or create a user or new repository. The file jauth.db write in a folder with 755 permission and I have the 500 error.

**Comment by virtusmankind** [comment]
I have the same error, but in error.log (in lizmap/var/log) there is this row at the end: 

2018-03-28 12:45:39	79.5.167.55	warning	2018-03-28 12:45:39	[2]	SQLite3::query(): Unable to prepare statement: 1, no such column: usr.status	/var/www/lizmap-web-client-master/lib/jelix/plugins/db/sqlite3/sqlite3.dbconnection.php	112

2018-03-28 12:45:39	79.5.167.55	error	2018-03-28 12:45:39	[403]	query non valida (no such column: usr.status (SELECT usr.usr_login as login, usr.usr_email as email, usr.usr_password as password, usr.firstname as firstname, usr.lastname as lastname, usr.organization as organization, usr.phonenumber as phonenumber, usr.street as street, usr.postcode as postcode, usr.city as city, usr.country as country, usr.comment as comment, usr.status as status, usr.keyactivate as keyactivate, usr.request_date as request_date, usr.create_date as create_date FROM jlx_user AS usr WHERE  usr.usr_login  = 'admin' LIMIT 0,1))	/var/www/lizmap-web-client-master/lib/jelix/plugins/db/sqlite3/sqlite3.dbconnection.php	115

What's wrong?

**Comment by omersiar** [comment]
Without touching the permissions restarting the virtual machine solved the problem for me.

**Comment by Gustry** [comment]
I think this can be closed.

Feel free to reopen if necessary
